src/core/common/logging.cpp

```diff
@@ -148,6 +148,42 @@ void otDump(otLogLevel aLogLevel, otLogRegion aLogRegion, const char *aId, const
     otLogDump("%s", buf);
 }
 
+#ifdef OPENTHREAD_CONFIG_LOG_PREPEND_LEVEL
+const char *otLogLevelToString(otLogLevel aLevel)
+{
+    const char *retval;
+
+    switch (aLevel)
+    {
+    case kLogLevelNone:
+        retval = "-NONE-";
+        break;
+
+    case kLogLevelCrit:
```

Comment:
I'm fine with both those suggestions. Does anyone else have any preferences here? I want to make sure everyone is happy. :)